shared/teams/team/custom-title/container.js

```diff
@@ -14,7 +14,7 @@ const mapStateToProps = (state: TypedState, {teamname}) => {
 }
 
 const mapDispatchToProps = (dispatch: Dispatch, {teamname}) => ({
-  onChat: () => dispatch(Chat2Gen.createPreviewConversation({teamname})),
+  onChat: () => dispatch(Chat2Gen.createPreviewConversation({teamname, reason: 'teamHeader'})),
```

Comment:
do we need different reasons for each place we dispatch this from?

Reply:
this is so we can understand the logs better and also we do different side effects depending on the reason